### PR TITLE
page designer

### PR DESCRIPTION
**Pull request by VelislavaC4Nexus**
### All pull requests:
* [ ] Have you considered security (e.g., XSS)?
* [ ] Have you considered desktop, mobile, and tablet form-factors?
* [ ] Have you considered [accessibility best practices](https://www.w3.org/WAI/standards-guidelines/wcag/)?

### Code
* [ ] Are the commits squashed into one commit?
* [ ] Is the [branch name and commit message](CONTRIBUTING.md#conventions-for-branch-names-and-commit-messages) following team convention?
* [ ] Is the code linted?
* [ ] Are all open issues, questions, and concerns by reviewers answered, resolved, and reviewed?

### Quality assurance
* [ ] If applicable, are unit tests written and are they passing?
* [ ] If applicable, are integration tests written and are they passing?
* [ ] Have checks linked in your pull request passed?

### Documentation
* [ ] Are there meaningful code comments included?

## Applicable to community contributors:
* [ ] Have you granted SFRA team access to your fork? [Grant access](CONTRIBUTING.md#community-contributors) 

### Documentation
* [ ] If applicable, is the change log updated?
* [ ] If applicable, have any UI text changes been reviewed by Documentation?
* [ ] If applicable, have any UI implementations been reviewed by the UX team?

### Security
* [ ] If applicable, has Security reviewed this code?
* [ ] If applicable, is a 3PP request submitted?

### CODE REVIEW

File: cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+var Template = require('dw/util/Template');
+var HashMap = require('dw/util/HashMap');
+/**
+ * Render logic for the component.
+ */
+module.exports.render = function (context) {
+   var model = new HashMap();
+   model.bannerMessage = context.content.alt;
+ return new Template('experience/components/commerce_assets/banner').render(model).text;
```
Comment (isergeev19) [inline cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.js]: Watch out for the indentations and formatting.

File: cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Banner",
+    "description": "A banner.",
+    "group": "commerce_assets",
+    "attribute_definition_groups": [
+      {
+       "id":"image",
+       "name":"Banner Image Configuration",
```
Comment (isergeev19) [inline cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.json]: Watch out for the indentations and formatting, even in json files

File: cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.js (hunk 1 above)
Comment (VelislavaC4Nexus) [reply cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.js]: Fixed

File: cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.json (hunk 2 above)
Comment (VelislavaC4Nexus) [reply cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.json]: Fixed

**Review by isergeev19 — APPROVED** [review]

File: cartridges/custom_cartridge_1/cartridge/templates/default/experience/components/commerce_assets/fbPost.isml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<div class="d-flex">
+   <div class="col-md-6">
+        <iframe src="${pdict.iframeSrc}" width="${pdict.width}" height="${pdict.height}" title="${pdict.title}"
+            style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowfullscreen="true"
```
Comment (isergeev19) [inline cartridges/custom_cartridge_1/cartridge/templates/default/experience/components/commerce_assets/fbPost.isml]: Lets not use inline styles, you can use css here

**Pull request merged**